extensions/leetcode/src/api.ts

```diff
@@ -61,3 +61,15 @@ query problem($titleSlug: String!) {
   }
 }
 `;
+
+export const getCodeSnippetsQuery = `
```

Comment:
Is it possible to add `codeSnippets` in  `searchProblemQuery` and `dailyChallengeQuery` directly so we only have to send one API instead of two?

Reply:
Thank you! That's a good point. I hadn't fully leveraged GraphQL's ability to request all needed fields in a single query. I've updated the `searchProblemQuery` and `dailyChallengeQuery` to include the `codeSnippets` field directly, eliminating the need for separate API calls.

This change:
1. Improves performance by reducing API calls by half
2. Simplifies the codebase

Thanks for helping me better utilize GraphQL's capabilities!

Reply:
Also, I wanted to check if this pr aligns with the milestones listed in the README.

Reply:
https://developers.raycast.com/api-reference/user-interface/icons-and-images

there are some icons... 
CodeBlock
Copy
Document or NewDocument
Code or CodeBlock
Snippets

It's quite difficult to choose one for new menu...

Reply:
> Also, I wanted to check if this pr aligns with the milestones listed in the README.

Yes, this PR aligns well, thanks!

> https://developers.raycast.com/api-reference/user-interface/icons-and-images
> 
> there are some icons... CodeBlock Copy Document or NewDocument Code or CodeBlock Snippets
> 
> It's quite difficult to choose one for new menu...

They all look great for me. I think you can go with whichever you prefer!